test/decode.js

```diff
@@ -0,0 +1,18 @@
+import chai, { expect } from 'chai'
+import chaiChange from 'chai-change'
+import { decode } from '../src/decode'
+
+chai.use( chaiChange )
+
+describe('decode', () => {
+  'use strict'
+
+  it('exists', () => {
+    expect( decode ).to.be.a( 'function' )
+  })
+
+  it('decodes emoji string to alphanumeric characters', () => {
+    expect(decode( '😭😅🐱' )).to.equal( 'w07' )
```

Comment:
This test only covers an example without spaces. Does this work the same as encode?